docs/core_docs/docs/integrations/document_loaders/web_loaders/hyperbrowser.mdx

```diff
@@ -0,0 +1,196 @@
+---
+sidebar_class_name: node-only
+---
+
+# Hyperbrowser
+
+## Overview
+
+The Hyperbrowser document loader provides a way to load web content into your LangChain application using Hyperbrowser's cloud browser infrastructure. It supports:
+
+- Single-page scraping
+- Multi-page crawling
+- Multiple output formats (markdown, HTML, links)
+- Advanced browser automation features
+
+| Class | Package |  Local | [PY support](https://python.langchain.com/docs/integrations/document_loaders/hyperbrowser/)|
+| :---: | :---: |  :---: |  :---: |
+| HyperbrowserLoader | langchain-js | ❌ | ✅ |
```

Comment:
`Package` should be `@langchain/community` - apologies but I don't think I have access to push to your branch to make changes myself.